tensorflow/compiler/mlir/tensorflow/utils/tf_xla_mlir_translate.cc

```diff
@@ -114,7 +114,7 @@ mlir::LogicalResult PrintHloModuleText(
 
   xla::HloModule* hlo_module = status_or_hlo_module.value().get();
 
-  output << hlo_module->ToString();
+  output << hlo_module->ToString(xla::HloPrintOptions());
```

Comment:
Nit, this avoid constructing a temporary value:
```suggestion
  output << hlo_module->ToString(xla::HloPrintOptions::Default());
```

Reply:
I have edited it to set two of the print options. Right now, those values are the default values, of those print options, but they are going to change after https://github.com/openxla/xla/pull/22800. Is this okay?